src/video/camerasource.cpp

```diff
@@ -214,14 +233,18 @@ void CameraSource::unsubscribe()
     }
     else
     {
+        //qDebug() << "second block";
         device->close();
         biglock = false;
     }
+    subscriptions--;
```

Comment:
Issues are correct, I'm not saying google is wrong. I'm saying it's not time and place to address them here. This code isn't my, I didn't write it, just updated it to current master. If you really want them fixed, make a PR or rise and issue about it. We used cppcheck before and it found several important issues, it was helpful, but using ++i or i++ in a situation where it happens once per videocall is pointless, ++i is more readable for me and I will stay with it. What I find more important, is to consider using clang in qtcreator, we would have [SCA on hands all the time](http://doc.qt.io/qtcreator/creator-clang-static-analyzer.html) and the same behaviour on all platforms. Last time I asked about it (over a year ago) most people were against this idea.

Fell free to make a PR like you did #2395 and fix those issues. This PR is not about nano optimizations that remove 3 CPU instructions from exec and I don't think it should be addressed here. Not fixing it. Stop wasting time...